scripts/dns-test-execute.bash

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# The integration tests call this to test Linux DNS control, using the `/etc/resolv.conf`
+# method which only works well inside Alpine Docker containers.
+
+set -euo pipefail
+
+HTTPBIN=test.httpbin.docker.local
+
+# Wait for client to ping httpbin (CIDR) resource through the gateway
+docker compose exec -it client timeout 60 \
```

Comment:
This all could be a lot easier to read if you extract `docker compose exec -it $container` into a function! :)

Something like:

```bash
function client() {
	docker compose exec -it client $1
}

function gateway() {
	docker compose exec -it gateway $1
}
```